samples/subsys/mgmt/hawkbit/README.rst

```diff
@@ -25,22 +25,22 @@ source code for this mode hawkbit_manual
 Caveats
 *******
 
-* The Zephyr port of ``Hawkbit`` is configured to run on a
+* The Zephyr port of ``hawkBit`` is configured to run on a
```

Comment:
I know you only did the search replace, but I don't think it makes sense to use double quotes here (and several other instances in this file)

<img width="603" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/128251/a3f6080e-cab6-4b95-8fd1-29e7f4ccf8eb">

Reply:
@ycsin would be great to address this as part of the clean-up?

Reply:
Yeah I forgot about this, will do another rev